syntax = "proto3";

package sbai_geo_path_publisher_protos;

import "sbai_geographic_protos/geo_path.proto";

message GeoPathWithRobotId {
    sbai_geographic_protos.GeoPath geo_path = 1;
    string robot_id = 2;
}
